### PR TITLE
feat(gui-v2): add `AuditTab`

### LINKED ISSUE #2733: vue3: Settings > Audit

### PR DESCRIPTION
**Pull request by mustafapc19**
## Change Summary

Fixes #2733 

Added `AuditTab` of settings modal. Also added loading logic.

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [x] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

image
image
image

### CODE REVIEW

**Comment by o1lab** [comment]
Would need a left nav drawer in this modal. 

Screenshot 2022-07-19 at 07 50 16

**Comment by mustafapc19** [comment]
@o1lab The ui being referred is not part of `AuditTab`. That is part of the modal, where I mounted `AuditTab` temporarily to show its UI in this PR. It will be independently developed later and will be integrated with this component.

**Comment by o1lab** [comment]
Cool

File: packages/nc-gui-v2/helpers/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import dayjs from 'dayjs'
```
Comment (pranavxc) [inline packages/nc-gui-v2/helpers/index.js:1]: you can move this to `dateTimeUtls.ts`

**Comment by o1lab** [comment]
@mustafapc19 : please see if this [double scrollbar issue](https://github.com/nocodb/nocodb/issues/2737) reported is managed in this PR.

**Comment by mustafapc19** [comment]
@o1lab This pr only has the inner component so does not have this problem. Will keep this in mind when working on modal integration.

**Review by pranavxc — APPROVED** [review]

**Pull request merged**